io/src/main/scala/sbt/nio/Glob.scala

```diff
@@ -123,10 +123,10 @@ object Glob extends LowPriorityGlobOps {
     }
     def \(component: String): Glob = this / component
     def glob(filter: FileFilter): Glob =
-      new GlobImpl(converter(repr), (0, 1), ConvertedFileFilter(filter))
+      new GlobImpl(converter(repr), (1, 1), ConvertedFileFilter(filter))
```

Comment:
Is this a semantic change from the PathFinder behavior?

Reply:
Yes and no. It is semantically different from 1.3.0-M2 when using `Glob` apis but is semantically identically when using PathFinder because we have:
```scala
  private[sbt] final class GlobPathFinder(val glob: Glob) extends PathFinder {
    override def get(): Seq[File] = {
      if (glob.range._1 == 0 && glob.range._2 == 0) {
        glob.base.toFile :: Nil
      } else if (glob.range._2 > 1) {
        val files = new java.util.LinkedHashSet[File].asScala
        **if (glob.pathFilter.accept(glob.base)) files.add(glob.base.toFile)**
        Path.defaultDescendantHandler(glob.base.toFile, glob.toFileFilter, files, glob.range._2)
        files.toIndexedSeq
      } else {
        Path.defaultChildHandler(glob.base.toFile, glob.toFileFilter)
      }
    }
  }
```
The line marked with ** was added to make `globRecursive` consistent with the old PathFinder api but the non recursive case never used the range of the glob anyway.